app/components/floating_text_input_label/index.tsx

```diff
@@ -224,7 +224,7 @@ const FloatingTextInput = forwardRef<FloatingTextInputRef, FloatingTextInputProp
         }
 
         return res;
-    }, [styles, theme, shouldShowError, focused, textInputStyle, focusedLabel, multiline, multilineInputHeight, editable]);
+    }, [styles, textInputStyle, multiline, multilineInputHeight]);
```

Comment:
IDE was complaining regarding dependencies. There are others like this throughout the PR.

Reply:
Do check the implications of each one of the removed ones

Reply:
My understanding is that we want to re-render on those cases, I'll add a comment so there is no warning and it is clear that is intentional.

On the other instances I only added dependencies that are in use within the block, so I believe they are ok. Let me know if you think otherwise :)

Reply:
Unless something weird is being done (and therefore, the problem is doing that weird thing) you should be able to trust the linter on any useMemo or useCallback. The dependencies stated there should be the ones you need.

With `useEffect` is more complicated, since the way `useEffect` is generally used, you don't always want to execute the effect when something inside changes. So those cases you have to look a bit more careful.